host/rpi/rpi.go

```diff
@@ -128,17 +128,37 @@ func (d *driver) Init() (bool, error) {
 	//
 	// This code is not futureproof, it will error out on a Raspberry Pi 4
 	// whenever it comes out.
+	// Revision codes from: http://elinux.org/RPi_HardwareHistory
 	rev := distro.CPUInfo()["Revision"]
 	if i, err := strconv.ParseInt(rev, 16, 32); err == nil {
 		// Ignore the overclock bit.
 		i &= 0xFFFFFF
-		if i < 0x20 {
+		switch i {
+		case 0x0002, 0x0003, // B v1.0
+			0x0004, 0x0005, 0x0006, // B v2.0
+			0x0007, 0x0008, 0x0009, // A v2.0
+			0x000d, 0x000e, 0x000f, // B v2.0
+			0x0010,  // B+ v1.0
+			0x0011,  // Compute Module 1
+			0x0012,  // A+ v1.1
+			0x0013,  // B+ v1.2
+			0x0014,  // Compute Module 1
+			0x0015,  // A+ v1.1
+			0x90021, // A+ v1.1
```

Comment:
This and the one below are the current version A and B, they should maybe be mapped as V2 as they have the 40 pin header.